courses/free-courses-en.md

```diff
@@ -246,6 +246,7 @@
 
 ### Cloud Computing
 
+* [Cloud Computing Full Course In 11 Hours](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka
```

Comment:
- Completing title
- Fix typo in platform/instructor? `Edureka` vs `Edureka!`

```suggestion
* [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
```

What do we think about?

Reply:
if you are agree, apply both suggestion

Reply:
Not resolved yet https://github.com/EbookFoundation/free-programming-books/pull/6581#discussion_r740621040 as Eric mention on https://github.com/EbookFoundation/free-programming-books/pull/6581#issuecomment-959191238

Reply:
Definitely resolved at https://github.com/EbookFoundation/free-programming-books/pull/6581/commits/2923ed28272358ef9095b40de50a7d10e4b77d79